src/style.css

```diff
@@ -1 +1,58 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-size: none;
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
You can use just - https://developer.mozilla.org/ru/docs/Web/CSS/background
And combine all these things